cms/templates/settings.html

```diff
@@ -347,7 +352,11 @@ <h2 class="title-2">${_("Introducing Your Course")}</h2>
                     </span>
 
                     <span class="msg msg-help">
-                    ${_("You can manage this image along with all of your other <a href='{}'>files &amp; uploads</a>").format(upload_asset_url)}
+                    ${Text(_("You can manage this image along with all of your other {files_uploads_html}")).format(
+                        files_uploads_html=HTML("<a href='{upload_asset_url}'>files & uploads</a>").format(
```

Comment:
I think there is a translation issue here though-- "files & uploads" (or I think it would actually be better to say "files and uploads") needs to be translated.